deepdives/build-process/README.md

```diff
@@ -0,0 +1,66 @@
+# Deep Dive: The Build Process
+
+## Why?
+As you start to create more and more ambitious apps, the time you invest into a build process which automates work for you, cleans up your code, and gives you access to more features becomes more and more worth it. These tools can be intimidating to learn at first, but with a bit of pre work they can make your life a ton easier.
+
+## The Parts of the Process
+
+### Compilers
+You've already been exposed to these with SASS and LESS compilers. You can also use compilers (or sometimes call transpilers) to convert modern es6 or es7 javascript code into widely supported es5 javascript. The most recent updates to javascript has helped to smooth over some ugly parts (I'm looking at your Javascript OOP) and added a ton of new features. We'll get into how to use these new features in the exercises. Luckily, as long as we are just considering vanilla javascript, as opposed to coffescript or typescript, the de facto standard for compiling your code is:
+
+##### Babel
+![Babel](https://avatars0.githubusercontent.com/u/9637642?v=3&s=400)
+
+Babel can convert basically any javascript (including jsx!) you can imagine into es5. It does this by requireing users to specify a ```preset``` or set of presets that babel will use to do the compilation. You can also specify ```plugins``` to extend the features of babel. The way that you specify these is different based on the rest of the tools we use. You can look at the examples above to see how it is done in different setups.
+
+###### Readings
+- [Docs](https://babeljs.io/)
+- [What is Babel the Javascript Compiler](http://www.programwitherik.com/what-is-babel-the-javascript-compiler/)
+
+### Bundlers
+You may have noticed that as the clientside portion or your apps get bigger dealling with importing scripts, css, and images into your pages becomes a huge pain in the @#$. Wouldn't it be nice if we could require files like we can on our server? Well do I have good news for you, you have two good options, browserify and webpack.
```

Comment:
You may have noticed.... -> You may have noticed that, as the clientside portions of your apps get bigger, dealing with importing scripts/CSS/images becomes a huge pain in the @#$.

Well do I have good news -> Well, do I have good news for you -- that's just what `browserify` and `webpack` let you do.